fragments/training/training-fragment.md

```diff
@@ -1,6 +1,68 @@
 # Model Training Fragment
 
-**THIS SECTION IS STILL UNDER DEVELOPMENT.**
+This document describes the structure and content of a Model Training object.
 
 | Field Name          | Type                | Description                                                           |
 |---------------------|---------------------|-----------------------------------------------------------------------|
+| `created_at`        | string              | **Required.** Approximate date and time at which the model was trained. It is formatted according to [RFC 3339, section 5.6].  |
+| `environment`       | [Environment]       | **Required.** Description of the environment used to train the model. |
+| `data`              | [string]            | **Required.** A list of URLs, each of which points to a [STAC Collection] representing source imagery used to train the model. |
+
+#### created_at
+
+This field is intended to capture the approximate date and time at which the model was trained in order to give 
+other developers a sense of the currency of the model. The model publisher is free to choose the exact event that 
+this timestamp represents (e.g. start of model training, completion of model training, etc.). Publishers may also 
+choose to use `00` for all time values if they wish to only represent a date (e.g.
+`2020-02-01T00:00:00Z`). This format keeps the spec in line the datetime format requirements from
+the STAC spec.
+
+#### data
+
+A list of URLs, each of which points to a [STAC Collection] representing input data used to train
```

Comment:
Got it, thanks. Yes, for unsupervised things get a bit tricky as the concept of validation is a bit different. 
This is one of those areas that after the first release we need to implement specific use cases and see how it can capture the metadata. 
Agreed on the approach to improve ML AOI Extension.